test/internal/xcschemes/write_schemes_tests.bzl

```diff
@@ -327,6 +332,9 @@ def write_schemes_test_suite(name):
             "--consolidation-maps",
             "some/consolidation_maps/0",
             "some/consolidation_maps/1",
+            # autogenerationModeExcludePatternsTargetName
+            "--autogeneration-mode-exclude-patterns-target-name",
+            ".*foo.*",
```

Comment:
Added this under the "Basic" test but happy to create a separate one if folks have concerns